app/assets/stylesheets/pages/announces_show.scss

```diff
@@ -1,3 +1,10 @@
+.announces_show_html {
+
+  .logo {
+    color:#3B5675;
```

Comment:
Color literals like `#3B5675` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#3B5675` should be written as `#3b5675`<br>Colon after property should be followed by one space